mrp_bom_configurable/models/__init__.py

```diff
@@ -0,0 +1,10 @@
+from . import input_config
+from . import input_line
+from . import input_constraint
+from . import mrp_bom
+from . import mrp_bom_line
+
+from odoo.tools import config
+
+if not config["without_demo"] or config["test_enable"]:
```

Comment:
we were using without_demo to test demo mode in several OCA/l10n-brazil modules and switch back to testing with a select demo from ir_module where... because it was more resilient to shity config values like empty vs False for without_demo as it was reported to us in several issues.

Reply:
Hi, do you have an example ?

Reply:
@bealdav @FranzPoize sorry I missed your message. So one way we fixed it is by looking if some demo record is loaded, for instance:
https://github.com/OCA/l10n-brazil/commit/a79caf2d211ef7efb84f7d913c137116a5851b05

and another way is looking at the demo column in the ir_module_module table:
https://github.com/OCA/l10n-brazil/commit/eece8ca1b84040699ef5658cfcb3f087c2276425#diff-50fe6815141d1f784405627ca86d85e38989b6be8ccc48e479f51480b94e27b1R28

again, simply using config["without_demo"] has real issues, we had to fix all the modules using this approach.